### PR TITLE
updated key_Alex file

### PR DESCRIPTION
**Pull request by AlexanderDawid**
My new change .. take it or not:)=

### CODE REVIEW

**Review by yohaglooko — CHANGES_REQUESTED** [review]
can you please update the comment thx

File: key_Alex [hunk 1]
```diff
@@ -1,2 +1,3 @@
 
 Generating a new SSH key for Alex
+Hello world!
```
Comment (AlexanderDawid) [inline key_Alex:3]: No.. master chief!

**Comment by yohaglooko** [comment]
ok i will do it for you.

**Comment by yohaglooko** [comment]
add to master

**Pull request merged**